src/components/BookList/BookList.js

```diff
@@ -1,30 +1,29 @@
-import React, { useState } from 'react';
-import { v4 as uuidv4 } from 'uuid';
+import React from 'react';
+import { useSelector } from 'react-redux';
+// import booksReducer from '../../redux/books/books';
```

Comment:
- [ ] Kindly remove all commented code from your codebase to make your codebase clean and readable 👍🏽